lib/lita/handlers/totems.rb

```diff
@@ -171,6 +172,11 @@ def add(response)
             # queue:
             queue_size = redis.rpush("totem/#{totem}/list", user_id)
             redis.hset("totem/#{totem}/waiting_since", user_id, Time.now.to_i)
+            current_owner_id = redis.get("totem/#{totem}/owning_user_id")
+            unless redis.sismember("user/#{current_owner_id}/totems/reminder", totem)
```

Comment:
nit: could abstract logic in a function like remindTotemHolderrr...im bad with names :)